tsl/src/dist_util.c

```diff
@@ -12,13 +12,16 @@
 #include <utils/fmgrprotos.h>
 
 #include "ts_catalog/catalog.h"
+#include "config.h"
 #include "dist_util.h"
 #include "errors.h"
 #include "funcapi.h"
 #include "loader/seclabel.h"
 #include "ts_catalog/metadata.h"
 #include "remote/dist_commands.h"
+#ifdef USE_TELEMETRY
 #include "telemetry/telemetry_metadata.h"
+#endif
 #include "utils/uuid.h"
 
 /*
```

Comment:
You should remove the double definition of:
```
/*
 * When added to a distributed database, this key in the metadata table will be set
 * to match the uuid (from ts_metadata_get_uuid()) of the access node.
 *
 * Therefore we can check if a database is the access node or not simply by
 * comparing the results of dist_util_get_id() and ts_metadata_get_uuid().
 */
#define METADATA_DISTRIBUTED_UUID_KEY_NAME "dist_uuid"
```

Reply:
Seems to be remains from a rebase. Removed.